examples/project/project.json

```diff
@@ -102,9 +102,18 @@
     },
     "RegionUsageRestrictions": {
       "Type": "String",
-      "AllowedValues": ["GOV_REGIONS_ONLY","COMMERCIAL_FEDRAMP_REGIONS_ONLY","NONE"],
+      "AllowedValues": [
+        "GOV_REGIONS_ONLY",
+        "COMMERCIAL_FEDRAMP_REGIONS_ONLY",
+        "NONE"
+      ],
       "Description": "Region usage restrictions that designate the project's AWS region",
       "Default": "NONE"
+    },
```

Comment:
I don't think using a map directly as a parameter is possible.
Therefore, chose to use `TagOwner` and set it on the `Resource.Tags.Owner` instead

Reply:
[q] not sure I understand this usage. What is the purpose of TagOwner could you please explain more?

Reply:
I couldn't find a way to specify a map to a CFN template.
Therefore, I created a `TagOwner` instead. 
From the user perspective, I couldn't think of any more valuable examples :/

Reply:
Doing some research on `tags` and `labels` in use, I could only find:

[The cluster example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/CLOUDP-243393_project_tags_add_attribute/examples/cluster/project-cluster.json#L73) doesn't expose a parameter, it sets the tag directly
  - uses `Environment=dev`
  
  @maastha do you have any suggestion or can we leave this as is? 😅

Reply:
Values used directly in [5fe061a](https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/1006/commits/5fe061a19422a20024a4b2ed2f504255d55ad68c)
Ok to resolve @maastha ?

Reply:
yes! thank you for looking into this!